anatomy/tracks/concept-exercises.md

```diff
@@ -363,10 +385,12 @@ class LasagnaTest < Minitest::Test
   end
 
   def test_preparation_time_in_minutes_with_one_layer
+    skip
```

Comment:
We indicated that all but the first test should be skipped, but the example didn't :)

Reply:
I think we're going to revert the skipping thing. Do we want to do that in this PR or a seperate one?

Reply:
I've added it to this PR.